SwiftReflector/SwiftInterfaceReflector/SwiftInterfaceReflector.cs

```diff
@@ -1645,16 +1654,21 @@ string TargetFromModuleFlags ()
 			return flagsValue;
 		}
 
+		static string [] ModulesThatWeCanSkip = new string [] {
+			"XamGlue",
+			"RegisterAccess",
+			"_StringProcessing",
+			"_Concurrenct",
+		};
```

Comment:
I'd move to a set, it will be better in terms of performance when doing the contains and will ensure that you do not have duplicates.